pages/getting-started/cloud-shell-quickstart.md

```diff
@@ -18,118 +18,87 @@ in under 30 minutes, all within your browser.
 
 Head over to [app.plural.sh](https://app.plural.sh) and set up an account.
 
-Directly navigate to [app.plural.sh/shell](https://app.plural.sh/shell) to access the Cloud Shell. You'll be prompted to OAuth with GitHub. We're just getting your permission to create a GitHub repository for Plural configuration on your behalf. Give your repository a name, being sure to select the right organization or individual account on the left.
+Navigate to [app.plural.sh/shell](https://app.plural.sh/shell) to access the Cloud Shell (if this is your first time using app.plural.sh, you should be automatically redirected to the onboarding flow). You'll be prompted to OAuth with either GitHub or GitLab. We're just getting your permission to create a repository for Plural configuration on your behalf. Give your repository a name, being sure to select the right organization or individual account in the first box. Your repository name must be unique within your account, which will be checked before you can advance to the next step.
 
-![](</assets/cloud-shell-quickstart/image-1.png>)
+![](</assets/cloud-shell-quickstart/github-config.png>)
 
 ### Set up a Cloud Provider
 
-You now have two options. If you're just trying out Plural, you can have a free demo cluster on the house that will last for 6 hours. Just keep in mind that this option will be only available once.
+Choose the option to set up your Cloud Provider in the browser. Plural can be set up on AWS, GCP, or Azure.
 
-![](</assets/cloud-shell-quickstart/image-2.png>)
+![](</assets/cloud-shell-quickstart/setup-options.png>)
 
-To set up with your own cloud provider, provide your credentials in the following screen.
+You can provide your credentials in the following screen.
 
 ### Set up a Workspace
 
 We'll now start setting up your Kubernetes cluster configuration. Here's a guide to the config:
 
-**cluster**: The name of your Kubernetes cluster. Name this based on what you're planning to run.
+**cluster**: The name of your Kubernetes cluster. Name this based on what you're planning to run. Feel free to use the provided default.
 
-**bucket prefix**: We provision S3 buckets for storing logs and state. Just enter any string to help us uniquely name your buckets.
+**bucket prefix**: We provision S3 buckets for storing logs and state. Enter any string to help us uniquely name your buckets, or use the provided default.
```

Comment:
I don't think we provide a default beyond the placeholder? Could be wrong here..

Reply:
Yeah I was going to ask about that - why don't we persist the placeholder? Are we just showing that it shouldn't just be plural, but not actually giving them the option to select that?

Reply:
Removed